WowUp.WPF/Utilities/StartupOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WowUp.Common.Enums;
+
+namespace WowUp.WPF.Utilities
+{
+    public class StartupOptions
+    {
+        [Option(shortName: 'i', longName: "install", HelpText = "Specify addon urls to install them")]
+        public IEnumerable<string> InputURLs { get; set; }
+        [Option(shortName: 's', longName: "silent", HelpText = "Start application minimized")]
+        public bool Silent { get; set; }
+        [Option(shortName: 'u', longName: "update", HelpText = "Update non-ignored addons on startup")]
+        public bool Update { get; set; }
+        [Option(shortName: 'f', longName: "force", HelpText = "Update all addons on startup")]
```

Comment:
Eh i could see someone complaining after running this and it breaking something.

Reply:
Well, the help text is there :) Also, I do not see a way to trigger this accidentally.